### DISCUSSION TITLE
Select with multiple choice feature

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [radix-ui/primitives#2398] [Help]
Will radix team add multiple choice feature to the Select element in the near future?

**Comment by AdsonCicilioti** [comment]
You can use `DropdownMenu` with `CheckboxItem's` for now. I also would like for this feature on `Select` component or an a new like a `Combobox`.

**Reply by benoitgrelard** [reply]
`DropdownMenu` + `CheckboxItem` is very different semantic so I wouldn't recommend it.
A `Combobox` would be the correct solution, we don't have one at the moment, but you could use one from another lib for now.
